### ISSUE TITLE
Crashes if GitHub API content is unknown: 'Cannot deserialize GitHub API content: unknown variant'

### ISSUE THREAD

**Issue by nicokosi** [nicokosi/pullpito#22]
The GitHub API [events](https://developer.github.com/v3/activity/events/) is used and its enumerations are encoded (issue types, statuses etc.). If enumerations are unknown, pullpito fails.
Example:
```sh
cargo run -- --repository nicokosi/pullpito
    Finished dev [unoptimized + debuginfo] target(s) in 0.22s
     Running `target/debug/pullpito --repository nicokosi/pullpito`
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: StringError("Cannot deserialize GitHub API content: unknown variant `added`, expected one of `closed`, `created`, `edited`, `opened`, `started` at line 1 column 11304") }', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
``` 
Either code all known enumerations, or find a way to ignore unknown ones.

**Comment by nicokosi** [comment]
I opened this `serde_json` issue: https://github.com/serde-rs/json/issues/470

**Comment by nicokosi** [comment]
Closed issue serde-rs/json#470 and opened issue serde-rs/serde#1353.

**Comment by nicokosi** [comment]
Fixed in https://github.com/nicokosi/pullpito/commit/0abb0317174c7175d89f228dfc37aa7a6a25bacf.
